data/json/items/tool_armor.json

```diff
@@ -1871,15 +1871,16 @@
         "pocket_type": "CONTAINER",
         "min_item_volume": "250 ml",
         "max_contains_volume": "1 L",
-        "max_contains_weight": "1 kg",
+        "max_contains_weight": "2 kg",
+		"max_item_lenght": "70 cm",
         "moves": 3,
         "flag_restriction": [ "SHEATH_KNIFE", "SHEATH_SWORD" ]
       }
     ],
     "use_action": [ { "type": "holster", "holster_prompt": "Sheath blade", "holster_msg": "You sheath your %s" }, "CROWBAR", "HAMMER" ],
     "qualities": [
       [ "HAMMER", 3 ],
-      [ "PRY", 1 ],
+      [ "PRY", 3 ],
```

Comment:
Survivor utility belt recipe contains hammer which has a prying quality of 1, not 3.

https://github.com/CleverRaven/Cataclysm-DDA/blob/27d6097612fccefbd66766189cb1aa1addfedd0d/data/json/recipes/armor/other.json#L352-L369

https://github.com/CleverRaven/Cataclysm-DDA/blob/1a292a90e0e4d4e2de64d7d0fd0125c3c436c1a5/data/json/items/tool/workshop.json#L287-L304

Reply:
Oh whoops. My bad!